### ISSUE TITLE
[age] allow custom paths to ssh files 

### ISSUE THREAD

**Issue by fionera** [gopasspw/gopass#2933]
### Summary
Currently it is not possible to use the age backend and point gopass to a specific key to use, e.g. when having a subdir in .ssh.

### Steps To Reproduce
1. Have a key somewhere not in ~/.ssh
2. Setup gopass with age

### Expected behavior
A flag to specify additional folders/directly select a key

### Environment
- OS: Mac OS X Sequoia
- OS version: Darwin 24.1.0 Darwin Kernel Version 24.1.0: Fri Aug  2 03:23:05 PDT 2024; root:xnu-11215.40.24.0.1~10/RELEASE_ARM64_T6000 arm64 Darwin
- gopass Version: gopass 1.15.13 go1.22.6 darwin arm64
- Installation method: Nix

**Comment by dominikschulz** [comment]
That sounds like something we should support. Good point.

**Comment by Dxuian** [comment]
is this still open?

**Comment by dominikschulz** [comment]
@Dxuian Yes, still open. Although I haven't thought through the implication of this. Feel free to give it a try, but if it unnecessarily complicates the implementation we might need to reject the request. Depends on how the implementation will end up.
